readme.md

```diff
@@ -26,27 +26,24 @@ A popular stance among the Sauerbraten community is "that's impossible", and thi
 ##### Different features are afoot
 giving a general but no inevitably defined idea of where we're going next
 
-* node.js scripting
- * (JavaScript + asynchronous + package manager)
+* Node.js scripting
+  * JavaScript + asynchronous + package manager
 * new user interface using HTML5 + CSS3
 * functionality to facilitate content-creating:
   * better editing features (e.g. a version control system for maps)
   * **extensibility** (e.g. for custom playermodels, sounds,..)
 * in-game content sharing
-* documentiation of the whole code base
 * refractored code to be more modern, scaleable, safe, non-blocking, modular, free, ... than before
  * modularity (makes working as/in a team easier)
```

Comment:
this is still malformatted

Reply:
I re-added it to the bullet point afterwards. Main thinking behind removing this line was: It's not a feature, really trival and should be a no-brainer.